environments/stg/global.tfvars

```diff
@@ -49,4 +49,14 @@ frontends = [
       SQLI = ["942440"]
     }
   },
+  {
+    name             = "pet-app1"
```

Comment:
shouldn't this be an actual app name? surely there's not one called pet-app1?

Reply:
That's the name of the app that has been spun up temporarily for testing purposes

Reply:
what are you testing?

Reply:
The different data flow to PET compared to the divorce app